### DISCUSSION TITLE
How to list share images in archive, next to each issue?

### DISCUSSION THREAD

**Discussion by christophberger** [buttondown/discussions#37] [Q&A]
Hi,

Is there a way to make the share images appear next to each newsletter issue in the archive? 

The archive list would look more pleasing, and people will find a particular issue easier if they remember its share image from social media.

**Comment by jmduke** [comment]
No, but we are tracking it over on https://github.com/buttondown/roadmap/issues/2391!

**Reply by christophberger** [reply]
Oo that's great to hear!

Thank you!
